bin/mailing-common/mailing-list-utils.rb

```diff
@@ -74,3 +74,27 @@ def query_subscribed_contacts(params)
     ALL.merge! results
   end
 end
+
+EMAIL = 'Email Address'
+NAME = 'Name'
+COUNTRY_CODE = 'CC'
+COUNTRY_CODE_US = 'US'
+
+def include_mailchimp_engineers(results)
```

Comment:
Agreed. ALL was leftover from the old scripts. I plan on improving / removing most of this, and coming up with a place to store the generated csv files :)

Yes, the numbers are the same.

* Before:
```
119666 hoc-engineers (initial query)
120655 including mailchimp engineers
120186 hoc-engineers (excluding volunteers)
```
* After:
```
119666 hoc-engineers (initial query, minus hoc-engineers-volunteers)
120186 including mailchimp engineers
```